src/components/Waveform/Waveform.js

```diff
@@ -211,6 +228,28 @@ export default class Waveform extends React.Component {
     return false;
   };
 
+  onZoomYPlus = (ev, step = 1) => {
+    let val = this.state.zoomY;
+
+    val = val + step;
+    if (val > MAX_ZOOM_Y) val = MAX_ZOOM_Y;
+
+    this.onChangeZoomY(val);
+    ev.preventDefault();
+    return false;
+  };
+
+  onZoomYMinus = (ev, step = 1) => {
```

Comment:
can be some currying wrapper to DRY; and you don't pass `step` anyway, so it can be +1/-1 from such wrapper:
`onClick={onZoomY(-1)}`